app/components/Snaps/SnapUIFooterButton/SnapUIFooterButton.tsx

```diff
@@ -135,6 +137,7 @@ export const SnapUIFooterButton: FunctionComponent<SnapUIFooterButtonProps> = ({
       size={ButtonSize.Lg}
       style={customButtonStyles.button}
       isDanger={snapVariant === 'destructive'}
+      testID={testID ?? (name ? `snap-footer-button-${name}` : undefined)}
```

Comment:
```suggestion
      testID={testID ?? `${name}-snap-footer-button`}
```
Let's mirror the extension